day1/part1.py

```diff
@@ -0,0 +1,13 @@
+def find_first_last_digit(line: str) -> str:
+    for character in line:
```

Comment:
There is the same code twice, maybe it would be better to adjust this to function first_digit and call it twice and add the results. 😄